cake-recipe.txt

```diff
@@ -7,7 +7,7 @@ Ingredients:
  - 1 teaspoon salt
  - 2 eggs
  - 1 cup milk
- - 1/2 cup vegetable oil
+ - 1/2 cup cannols oil
```

Comment:
Looks like your commit message canols and ingredient cannols are misspelled! please correct